### DISCUSSION TITLE
BUILD_URL doesn't work for GITLAB

### DISCUSSION THREAD

**Discussion by VladyslavZagorodnyi** [qa-guru/allure-notifications#114] [Q&A]
Hello, I configured Slack  reporting for Gitlab. Everything is fine apart from link for full allure report (BUILD_URL) : 
![image](https://user-images.githubusercontent.com/29700638/179395213-cea6d075-17b2-45a1-a761-dbe6a90d06e0.png)

it is actual Slack message that I receive:
![image](https://user-images.githubusercontent.com/29700638/179395526-7610376d-61dd-48df-9d72-c4079f445fd6.png)
ISSUE:
"Report available by link" has no link

--------
Here you can see content of config json file
![image](https://user-images.githubusercontent.com/29700638/179395910-84afdff1-fb4b-4df7-986e-c0d7f00885dd.png)

also 

after_script (because Gitlab) that I use 
![image](https://user-images.githubusercontent.com/29700638/179395919-ebb028f7-1384-4522-aede-145a27c52f51.png)

------
How to make (BUILD_URL) to generate actual link on Gitlab, please suggest?

**Comment by VladyslavZagorodnyi** [comment]
@svasenkov , @kadehar , @valfirst , @igorsoloviov , @ferras777 , @Konyaz , @Gorbatenko , @bookovna , @michael-horbachov , @MRC81 , @dandreyanov FYI

**Comment by valfirst** [comment]
There is no predefined variable `BUILD_URL` in GitLab, see the list of predefined variables here : https://docs.gitlab.com/ee/ci/variables/predefined_variables.html. 

Most likely it's needed to use `CI_JOB_URL` or `CI_PIPELINE_URL`
